uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/toil-request.controller.js

```diff
@@ -134,17 +130,17 @@ define([
      * @return {Promise}
      */
     function getReferenceDate () {
```

Comment:
```js
function getReferenceDate () {
  var isMultipleDays = parentCtrl.uiOptions.multipleDays;

  return getReferenceDateForField({
    hasErrors: isMultipleDays
      ? !parentCtrl.request.to_date && !parentCtrl.request.from_date
      : !parentCtrl.request.from_date;,
    label: isMultipleDays ? 'To Date' : 'From Date',
    value: isMultipleDays ? parentCtrl.request.to_date : parentCtrl.request.to_date
  });
}
```
?

Reply:
I haven't refactored these functions. But you proposal fits perfectly, thank you.